docs/remote/ssh-tutorial.md

```diff
@@ -99,6 +99,12 @@ The `user` is the username you set when adding the SSH public key to your VM. Fo
 
 ![Virtual machine public IP address](images/ssh-tutorial/vm-public-ip-address.png)
 
+Before connecting in Remote - SSH, you can verify you're able to connect to your VM via a command prompt using `ssh user@hostname`.
+
+> Note: If you run into an error `ssh: connect to host <host ip> port 22: Connection timed out`, you may need to delete NRMS-Rule-106 from the Networking tab of your VM:
+
+![Virtual machine list of NRMS rules](images/ssh-tutorial/vm-nrms-rules.png)
```

Comment:
This image doesn't clearly show NRMS-Rule-106. Perhaps it is the entry with the SSH warning triangle?
Can you add a red outline (3 pixels is the convention) to highlight on the entry to remove?
Also since this image is for the Note, you can indent 3 spaces to indent the image.

Reply:
I realized that I included an image from a VM where I had already deleted Rule 106, so you're absolutely right that the image didn't clearly show NRMS-Rule-106! I added a red outline around the name of the rule and the prompt to delete it. I used paint to do this, and the 3 pixel border looks a bit smaller than the other red rectangles I see in tutorials, so let me know if I should make it more bold, or if you'd like me to make any other adjustments to this image (i.e. cropping it more, highlighting some other component of it with the red rectangle).

Also, I recognize the VM I had on-hand with a Rule 106 available has a different name/IP address than Sana's VM we reference in other parts of the tutorial, but since this screenshot is to emphasize Rule 106, I figured that would be alright. If you think I should crop or erase the name and CPU, or make a new VM with a different name, please let me know.

I added the 3 space indentation as well.

Reply:
Looks great. Thanks for redoing the image.